### DISCUSSION TITLE
PrivacyBadger@Chrome@Linux consuming high CPU - how does one debug (to see if it is related to a specific tab)?

### DISCUSSION THREAD

**Discussion by nhed** [EFForg/privacybadger#2953] [Q&A]
Seen it happen several times (over several PB versions).  Laptop fans spinning up, `htop` showing high cpu taken by chrome, chrome's task manager shows PB as highest CPU user.

![image](https://github.com/EFForg/privacybadger/assets/513759/13cd85fc-a7c1-4549-b881-4bcdc9565feb)

- `uname -r`:  `6.5.10-200.fc38.x86_64`
- `rpm -q google-chrome-stable`:  `google-chrome-stable-120.0.6099.224-1.x86_64`
- PB version: `2024.2.6`

Disabled & Re-enabled and things quieted down

Did not see any discussion re CPU usage (is this the right place to have this convo?)

**Comment by ghostwords** [comment]
Hi @nhed, thanks for getting in touch! Let's continue this in https://github.com/EFForg/privacybadger/issues/2954.
